### PR TITLE
feat(build): Upgrade to puppeteer@7.1.0

### PR DESCRIPTION
**Pull request by fuhlig**
- use current puppeteer version
- breaking changes introduced in puppeteer v6 or v7 do not affect BackstopJS
- tests successful

### CODE REVIEW

File: remote/index.js [hunk 1]
```diff
@@ -108,9 +108,7 @@ module.exports = function (app) {
       });
 
       const reportConfigFilename = path.join(
-        PROJECT_PATH,
-        'backstop_data',
-        'html_report',
+        _config.paths.html_report,
```
Comment (garris) [inline remote/index.js:111]: @fuhlig   Sorry -- is this change deliberate?   Not sure this looks right.

**Review by garris — CHANGES_REQUESTED** [review]
Thanks for the PR updating puppeteer.  Please just let me know about the other comment.  Cheers.

File: remote/index.js (hunk 1 above)
Comment (fuhlig) [reply remote/index.js:111]: Oops, leftover. Will drop the commit

Comment (fuhlig) [reply remote/index.js]: Dropped the commit.
But while you are at it - there is a pending PR for this exact change ;)

#1172 I would appreciate if you could take a look. Thanks @garris

**Review by garris — APPROVED** [review]
Thank you!

File: remote/index.js (hunk 1 above)
Comment (garris) [reply remote/index.js]: Will 👀

**Comment by schrufygroovy** [comment]
@garris @fuhlig  was this changed from `dependencies` to `devDependencies` on purpose? could not find any explanation

**Comment by fuhlig** [comment]
> @garris @fuhlig was this changed from `dependencies` to `devDependencies` on purpose? could not find any explanation

The version got upgraded within this PR, not moved to another dependency category. 
Not sure how it affects the usage of backstop is puppeteer is in `dependencies` or `devDependencies`.

**Comment by schrufygroovy** [comment]
@fuhlig check the diff of this pullrequest: https://github.com/garris/BackstopJS/pull/1284/files
Looks clearly like moving to other dependency category for me.

Followup projects must include `puppeteer` then themself --> breaking them.

**Comment by fuhlig** [comment]
Totally overlooked that - it is on me 🤦‍♂️ 
Sorry about that!
Will create a PR to fix it

**Comment by fuhlig** [comment]
#1286
